packages/components/src/TagGroup/TagGroup.tsx

```diff
@@ -2,27 +2,34 @@ import type RAC from 'react-aria-components';
 import { TagGroup, TagList, TagListProps } from 'react-aria-components';
 import { WidthProp, useClassNames } from '@marigold/system';
 import { FieldBase, FieldBaseProps } from '../FieldBase/FieldBase';
+import { TagGroupHiddenInput } from './TagGroupHiddenInput';
 
 // Props
 // ---------------
 type RemovedProps = 'className' | 'style' | 'children' | 'isRequired';
 
 export interface TagGroupProps
   extends Omit<RAC.TagGroupProps, RemovedProps>,
-    Pick<TagListProps<object>, 'items' | 'children' | 'renderEmptyState'>,
+    Pick<TagListProps<object>, 'items' | 'children'>,
     Pick<FieldBaseProps<'label'>, 'label' | 'description'> {
   variant?: string;
   size?: string;
+
   /**
    * Sets the width of the field. You can see allowed tokens here: https://tailwindcss.com/docs/width
    * @default full
    */
   width?: WidthProp['width'];
+
+  /**
+   * The name of the field, used when submitting form data.
+   */
+  name?: string;
+
   /**
-   * Displays a remove button on each tag.
-   * @default false
+   * Provides content to display when there are no items in the tag list.
    */
-  allowsRemoving?: boolean;
```

Comment:
isn't this than also a breaking change if you remove this prop?

Reply:
well... it actually did nothing :D not sure when. But there is no "allowsRemoving" on RACs `TagGroup` component

Reply:
okay but maybe its used by someone?

Reply:
okay, I found *one* usage of the prop in our apps (but it doesn't do anything since months...). Do you want me to tag this as a breaking change? I would categorize it as a bug 🙈